core/src/main/resources/nitf_spec.xml

```diff
@@ -1717,6 +1717,16 @@
         <field length="1" fixed_value="0"/>
     </tre>
 
+    <tre name="SECURA">
+        <field name="FDATTIM" length="14" type="string"/>
+        <field name="NITFVER" length="9" type="string"/>
+        <field name="NFSECFLDS" length="207" type="string"/>
+        <field name="SECSTD" length="8" type="string"/>
+        <field name="SECCOMP" length="8" type="string"/>
+        <field name="SECLEN" length="5" type="integer"/>
+        <field name="SECURITY" length_var="SECLEN" type="binary"/>
```

Comment:
`binary` isn't an allowed value for the schema. We could add it (to the schema, and the parsing), or just use `string` (which is what I did for CCINFA).

Reply:
strung it.